collectors/proc.plugin/proc_net_wireless.c

```diff
@@ -0,0 +1,501 @@
+
+		if(timercmp(&current->updated, timestamp, <)) {
+
+			struct netwireless *to_free = current;
+			current = current->next;
+			netwireless_free(to_free);
+
+			if(previous) {
+				previous->next = current;
+			}
+			else {
+				netwireless_root = current;
+			}
+
+		}
+		else {
+			previous = current;
+			current = current->next;
+		}
+
+    }
+}
+
+
+
+// finds an existing interface or creates a new entry
+static struct netwireless *find_or_create_wireless(const char *name) {
+    struct netwireless *wireless;
+
+    uint32_t hash = simple_hash(name);
+
+    // search it, from begining to the end
+    for(wireless = netwireless_root ; wireless ; wireless = wireless->next) {
+        if(unlikely(hash == wireless->hash && !strcmp(name, wireless->name))) {
+            return wireless;
+        }
+    }
+
+    // create a new one
+	wireless = callocz(1, sizeof(struct netwireless));
+	wireless->name = strdupz(name);
+	wireless->hash = simple_hash(name);
+	wireless->next = NULL;
+
+    // link it to the end
+    if(netwireless_root) {
+        struct netwireless *last_node;
+        for(last_node = netwireless_root; last_node->next ; last_node = last_node->next);
+        last_node->next = wireless;
+    }
+    else
+        netwireless_root = wireless;
+
+    return wireless;
+}
+
+static bool configure_device(int do_status, int do_quality, int do_discarded_packets, int do_missed,
+							 struct netwireless *wireless_dev) {
+	wireless_dev->do_status = do_status;
+	wireless_dev->do_quality = do_quality;
+	wireless_dev->do_discarded_packets = do_discarded_packets;
+	wireless_dev->do_missed_beacon = do_missed;
+	wireless_dev->configured = true;
+
+	char buffer[RRD_ID_LENGTH_MAX + 1];
+
+	snprintfz(buffer, RRD_ID_LENGTH_MAX, "%s", wireless_dev->name);
+	wireless_dev->chart_id_net_status = strdupz(buffer);
+
+	snprintfz(buffer, RRD_ID_LENGTH_MAX, "%s", wireless_dev->name);
+	wireless_dev->chart_id_net_link = strdupz(buffer);
+
+	snprintfz(buffer, RRD_ID_LENGTH_MAX, "%s", wireless_dev->name);
+	wireless_dev->chart_id_net_level = strdupz(buffer);
+
+	snprintfz(buffer, RRD_ID_LENGTH_MAX, "%s", wireless_dev->name);
+	wireless_dev->chart_id_net_noise = strdupz(buffer);
+
+	snprintfz(buffer, RRD_ID_LENGTH_MAX, "%s", wireless_dev->name);
+	wireless_dev->chart_id_net_discarded_packets = strdupz(buffer);
+
+	snprintfz(buffer, RRD_ID_LENGTH_MAX, "%s", wireless_dev->name);
+	wireless_dev->chart_id_net_missed_beacon = strdupz(buffer);
+
+	return true;
+}
+
+int do_proc_net_wireless(int update_every, usec_t dt) {
+	(void)dt;
+	static procfile *ff = NULL;
+	static int do_status = -1, do_quality = -1, do_discarded_packets = -1, do_missed = -1;
+	static unsigned long long int dt_to_refresh_speed = 0;
+	static char *proc_net_wireless_filename = NULL;
+	static int enable_new_interfaces = -1;
+
+	if(unlikely(enable_new_interfaces == -1)) {
+		char filename[FILENAME_MAX + 1];
+		snprintfz(filename, FILENAME_MAX, "%s", "/proc/net/wireless");
```

Comment:
You have to use `netdata_configured_host_prefix` like it is used in https://github.com/netdata/netdata/blob/master/collectors/proc.plugin/proc_meminfo.c#L123. It is needed for monitoring a host from a containerized Netdata.

Reply:
I already fixed this with new PR.